src/core/components/cards/cardCollapsible/cardCollapsible.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { Collapsible, type ICollapsibleProps } from '../../collapsible';
+import { Card } from '../card';
+
+export type CollapsedSize = 'sm' | 'md' | 'lg';
+
+export interface ICollapsibleCardProps extends Omit<ICollapsibleProps, 'buttonVariant'> {}
+
+export const CardCollapsible: React.FC<ICollapsibleCardProps> = (props) => {
+    const {
+        collapsedSize,
+        children,
+        className,
+        buttonLabelOpened,
+        buttonLabelClosed,
+        defaultOpen,
+        customCollapsedSize,
+        ...otherProps
+    } = props;
+
+    const [isOpen, setIsOpen] = useState(defaultOpen);
+
+    const handleToggle = (toggle: boolean) => {
+        setIsOpen(toggle);
+    };
+
+    const cardCollapsibleClassName = classNames('relative px-4 pt-4 transition-all duration-300 md:px-6 md:pt-6');
+    const blinderClassNames = classNames(
```

Comment:
I'm not sure about this because we're setting a property on the API that leads to believe that the `Collapsible` has a blinder/overlay, but based off of the implementation, it really does not. It's the `CardCollapsible` that does. Perhaps overriding my previous comment the `blinderClassName` prop should be more like footer classnames that this specific component just uses to render an overlay. Wdyt?

Reply:
That makes sense as `footerClassName` I think re: broadest interpretation for reusability.